scripts/generate_schemas.py

```diff
@@ -200,32 +204,63 @@ def __init__(self, version, schemaDir, outputFile):
         self.schemaDir = schemaDir
         self.outputFile = outputFile
         self.classes = []
-        for f in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
-            self.classes.append(SchemaClass(f))
+        for avscFile in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
+            self.classes.append(SchemaClass(avscFile))
+        self.requestClassNames = [
+            cls.name for cls in self.classes
+            if re.search('Search.*Request', cls.name)]
```

Comment:
Is there a reason these are 'Search.*' vs 'Search.+'?
Not really important as long as we never have anything called SearchRequest, but still.